features/networking/sdn.feature

```diff
@@ -66,53 +66,68 @@ Feature: SDN related networking scenarios
   @admin
   @destructive
   Scenario: SDN will detect the version and plugin type mismatch in openflow and restart node automatically
-    Given the master version >= "3.7"
-    And I select a random node's host
-    Given the cluster network plugin type and version and stored in the clipboard
-    And system verification steps are used:
-    """
-    When I run ovs dump flows commands on the host
-    Then the step should succeed
-    Then the output should contain "<%= cb.net_plugin[:type] %>.<%= cb.net_plugin[:version] %>"
-    """
-    And the node service is verified
-    And the node network is verified
-
-    When I run the ovs commands on the host:
-      | ovs-ofctl -O openflow13 mod-flows br0 "table=253, actions=note:<%= cb.net_plugin[:type] %>.ff" |
-    Then the step should succeed
+    Given I select a random node's host
+    And evaluation of `node.name` is stored in the :node_name clipboard
+
+    #Capturing the node's corresponding sdn pod to execute ovs commands on it
+    When I run the :get admin command with:
+       | resource      | pods                              |
+       | fieldSelector | spec.nodeName=<%= cb.node_name %> |
+       | namespace     | openshift-sdn                     |
+       | l             | app=sdn                           |
+       | output        | json                              |
+    Then the step should succeed
+    And evaluation of `@result[:parsed]['items'][0]['metadata']['name']` is stored in the :sdn_pod clipboard
```

Comment:
In #206 I made step `I run command on the "<%= cb.node_name %>" node's sdn pod:` also cache the pod. Could you:
* rebase this pull request
* remove the `get` and `clipboard` steps above
* instead of `cb.sdn_pod` in scenario use `pod.name`

This will make scenario shorter and nicer.

Reply:
Other than that looks good.

Reply:
Sounds good, I will look at that. Thanks!

Reply:
Done! Thanks for #206 @akostadinov